test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+describe RecipesController do
+  it "should get index" do
+      VCR.use_cassette("recipes") do
+        search_term = "chicken"
+        response = EdamamApiWrapper.list_recipes(search_term)
```

Comment:
This is a **controller** test and yet you are testing the wrapper!

You're also not passing in the search term with the `get` request.